src/js/api/config.js

```diff
@@ -122,6 +123,9 @@ define([
             ]);
 
             config.playlist = [ obj ];
+        } else if (_.isArray(configPlaylist.playlist)) {
+            config.feedData = configPlaylist;
+            config.playlist = configPlaylist.playlist;
```

Comment:
@robwalch  if config.feedData is equal to configPlaylist, should configPlaylist be renamed? It seems confusing that config.playlist would be configPlaylist.playlist